common/src/main/java/com/mx/path/core/common/exception/PathRequestException.java

```diff
@@ -70,6 +70,12 @@
  * <p>Example:
  *
  * <pre>{@code
+ * public class FailedLoginException extends AccessorUserException {
+ *   public FailedLoginException(String message, String userMessage) {
+ *     super(message)
+ *   }
+ * }
+ *
  *     throw new AccessorUserException("Identity unknown", "Username and/or password did not match system", PathResponseStatus.UNAUTHORIZED)
  *       .withReason("Bad password")
  *       .withCode("1401");
```

Comment:
I'm a little confused by this example. Should `throw new AccessorUserException` be changed to `throw new FailedLoginException`?

Reply:
Fixed.